### PR TITLE
solution

### PR DESCRIPTION
**Pull request by liza-strykharchuk**

### CODE REVIEW

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,20 @@
...
+
+  if (days < 3) {
+    return result;
+  }
+
+  if (days < 7) {
+    return result - twenty;
+  }
+
+  return result - fifty;
```
Comment (Viktor-Kost) [inline src/calculateRentalCost.js:44]: it'd be better to exchange like this
```suggestion
  if (days >= 7) {
    return rentalCost - proDiscount;
  }

  if (days >= 3) {
    return rentalCost - baseDiscount;
  }

  return rentalCost;
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Good job!
Looks much better. 
Some minor fixed needed

**Review by Viktor-Kost — APPROVED** [review]
Perfect!

**Review by nazarmatsevych — APPROVED** [review]

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
* test

**Pull request closed without merging**